machineSetup/setupPerfMachine.ps1

```diff
@@ -0,0 +1,144 @@
+﻿#Set power scheme to high performance
+powercfg /S 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c
```

Comment:
I will suggest to check for $LASTEXITCODE, in every step, and exit script as soon as there is a failure.